zrpc/internal/client.go

```diff
@@ -190,3 +204,19 @@ func WithUnaryClientInterceptor(interceptor grpc.UnaryClientInterceptor) ClientO
 			grpc.WithChainUnaryInterceptor(interceptor))
 	}
 }
+
+// WithRetryPolicy configuration grpc.ServiceConfig.methodConfig
+// if retry = true, policy = empty // use defaultRetryPolicy
+// if retry = true, policy = "some_config" // use some_config and defaultRetryPolicy
+// if retry = false // don't use retryPolicy。
+func WithRetryPolicy(retry bool, policy ...string) ClientOption {
```

Comment:
IMO using `WithRetryPolicy(retry bool, policy string)` is more clarity. 
I understand that you use variadic args to make policy an optional parameter, but when the user passes in multiple values, there is no effect and no error message, which increases the cost of understanding.